extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -217,7 +217,8 @@ export class GoogleSubscriptionsPlatform {
 
   /** @override */
   getEntitlements() {
-    return this.runtime_.getEntitlements().then(swgEntitlements => {
+    return this.runtime_.getEntitlements(this.serviceAdapter_
+        .getEncryptedDocumentKey()).then(swgEntitlements => {
```

Comment:
This call requires `serviceId`, right? E.g. you need to pass `google.com` here?

Reply:
done